README.md

```diff
@@ -10,8 +10,8 @@ This is a Slack bot that publishes a team's pull requests to their Slack Channel
 
 ## How to use it?
 
-### Config file
-Fork the repo and add/change the config file that relates to your github organisation. For example, the alphagov config file is located at [config/alphagov.yml](https://github.com/binaryberry/seal/blob/master/config/alphagov.yml).
+### Config
+Fork the repo and add/change the config files that relate to your github organisation. For example, the alphagov config file is located at [config/alphagov.yml](https://github.com/binaryberry/seal/blob/master/config/alphagov.yml) and the config for scheduled daily visits can be found in [bin](https://github.com/binaryberry/seal/tree/master/bin)
```

Comment:
Link should probably be relative rather than absolute

Reply:
Yup, happy to approve once that's done